joblib/_store_backends.py

```diff
@@ -292,16 +294,29 @@ def reduce_store_size(self, bytes_limit):
                 # the folder already.
                 pass
 
-    def _get_items_to_delete(self, bytes_limit):
-        """Get items to delete to keep the store under a size limit."""
+    def _get_items_to_delete(self, bytes_limit, files_limit=None):
```

Comment:
couldn't `bytes_limit` be made into `bytes_limit=None` and thus optional? 
Also in https://github.com/joblib/joblib/issues/1183 I contemplated  that it should be possible to "garbage collect" based on `last_accessed`, so may be `age_limit=None` with corresponding handling could be added?

Reply:
@yarikoptic The new signature is the way it is in order to provide backwards compatibility with old invocations.  If it weren't for that, there wouldn't even be a point to making `files_limit` optional.

Reply:
`age_limit` added.